src/views/Home/components/LostAndFoundCard/LostAndFoundCard.module.scss

```diff
@@ -0,0 +1,4 @@
+.linkHeader {
+  cursor: pointer;
+  text-decoration: underline;
```

Comment:
This is leading to the Lost and Found card title being underlined, not sure if we want this. Otherwise, LGTM!

Reply:
Changed to only underline on hover.